openedx_lti_tool_plugin/tasks.py

```diff
@@ -0,0 +1,83 @@
+"""Tasks for openedx_lti_tool_plugin."""
+import logging
+from datetime import datetime, timezone
+
+from celery import shared_task
+from django.contrib.auth import get_user_model
+from opaque_keys.edx.keys import CourseKey, UsageKey
+
+from openedx_lti_tool_plugin.edxapp_wrapper.grades_module import course_grade_factory
+from openedx_lti_tool_plugin.edxapp_wrapper.modulestore_module import modulestore
+from openedx_lti_tool_plugin.models import LtiGradedResource
+
+log = logging.getLogger(__name__)
+
+
+@shared_task(name='openedx_lti_tool_plugin.tasks.send_problem_score_update',)
+def send_problem_score_update(
+    problem_weighted_earned: str,
+    problem_weighted_possible: str,
+    user_id: str,
+    problem_id: str
+):
+    """Send problem score update task.
+
+    Task to update the AGS score of a problem asynchronously.
+
+    Args:
+        problem_weighted_earned: Grade earned for the problem.
+        problem_weighted_possible: Grade possible for the problem.
+        user_id: Grading user id.
+        problem_id: Problem id.
+    """
+    for graded_resource in LtiGradedResource.objects.all_from_user_id(user_id=user_id, context_key=problem_id):
+        log.info('LTI AGS: Sending AGS update for problem %s with user %s', problem_id, user_id)
+        graded_resource.update_score(
+            problem_weighted_earned,
+            problem_weighted_possible,
+            datetime.now(tz=timezone.utc)
```

Comment:
```suggestion
            datetime.now(tz=timezone.utc),
```
@alexjmpb Trailing comma